go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.1.2
 	github.com/hashicorp/go-azure-helpers v0.55.0
-	github.com/hashicorp/go-azure-sdk v0.20230306.1165107
+	github.com/hashicorp/go-azure-sdk v0.20230307.1120859
```

Comment:
there's some base layer changes included here, might be worth rebasing this once #20810 has been merged?

Reply:
Gotcha, will do 👍